protocols/118e8c/118e8c.ot2.apiv2.py

```diff
@@ -0,0 +1,184 @@
+import csv
+import os
+
+metadata = {
+    'protocolName': 'FluoGene HLA NX 96-Well Setup',
+    'author': 'Steve <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request',
+    'apiLevel': '2.9'
+}
+
+
+def run(ctx):
+
+    [dispense_height, aspirate_height, touch_radius, touch_v_offset,
+     touch_speed, tip_max, dna_volume,
+     reservoir_fill_volume] = get_values(  # noqa: F821
+        "dispense_height", "aspirate_height", "touch_radius", "touch_v_offset",
+        "touch_speed", "tip_max", "dna_volume", "reservoir_fill_volume")
+
+    ctx.set_rail_lights(True)
+
+    """
+    for reservoir column tracking between protocol runs
+    """
+    # for reservoir column tracking between protocol runs
+    if not ctx.is_simulating():
+        file_path = 'temporary/columnandtiptracking.csv'
+        file_dir = os.path.dirname(file_path)
+        # check for file directory
+        if not os.path.exists(file_dir):
+            os.makedirs(file_dir)
+        # check for file; if not there, create initial csv
+        if not os.path.isfile(file_path):
+            with open(file_path, 'w') as outfile:
+                outfile.write(",".join([
+                 "0", "A1 of Opentrons 96 Filter Tip Rack 20 µL on 10",
+                 "A1 of Opentrons 96 Filter Tip Rack 200 µL on 11", "\n"]))
+
+    current_data_list = []
+    if ctx.is_simulating():
+        current_data_list = [0,
```

Comment:
I'll add this comment here, but I think there should be a parameter added that will reset these stored values (in the case that the user wants to start with a fresh setup)